csharp/Adapter/Microsoft.Spark.CSharp/Network/ByteBufChunk.cs

```diff
@@ -286,6 +288,7 @@ private void Dispose(bool disposing)
                 return;
             }
 
+            logger.LogDebug("Disposing ByteBufChunk [{0}].", RuntimeHelpers.GetHashCode(this).ToString("X"));
```

Comment:
Changed to the below:

logger.LogDebug("Disposing ByteBufChunk [{0}].", ToString());